framework/quota/src/main/java/org/apache/cloudstack/quota/vo/ServiceOfferingVO.java

```diff
@@ -149,6 +149,7 @@ public ServiceOfferingVO(ServiceOfferingVO offering) {
         vmType = offering.getSystemVmType();
         systemUse = offering.isSystemUse();
         dynamicScalingEnabled = offering.isDynamicScalingEnabled();
+        diskOfferingStrictness = offering.diskOfferingStrictness;
```

Comment:
I think this file should be removed and not maintained, It pertains to the same table as https://github.com/apache/cloudstack/pull/7398/files#diff-bbbf939d866dbaec16b3c9ce76664ab203f7ba2a914e29772f795dd44a2cf993 and does not add value.

Reply:
@DaanHoogland will create a cleanup PR for it

Reply:
@DaanHoogland created a PR here, https://github.com/apache/cloudstack/pull/7439